tests/System/Extension/ClassMover/Command/ClassMoveCommandTest.php

```diff
@@ -26,14 +26,15 @@ public function testSmokeSuccess(string $command, array $fileMap): void
         $this->assertSuccess($process);
 
         foreach ($fileMap as $filePath => $shouldExist) {
-            $exists = file_exists($this->workspaceDir() . '/' . $filePath);
+            $absFilePath = $this->workspaceDir() . '/' . $filePath;
 
             if ($shouldExist) {
-                $this->assertTrue($exists);
+                $this->assertFileExists($absFilePath);
+                $this->assertTrue(is_file($absFilePath));
                 continue;
             }
 
-            $this->assertFalse($exists);
+            $this->assertFileDoesNotExist($absFilePath);
```

Comment:
Using better assertions, those are much more helpful (it shows the path that is or isn't there).